include/GafferImage/ImageReader.h

```diff
@@ -110,10 +110,18 @@ class ImageReader : public ImageNode
 		Gaffer::StringPlug *colorSpacePlug();
 		const Gaffer::StringPlug *colorSpacePlug() const;
 
+		Gaffer::StringPlug *colorSpaceMetadataNamePlug();
```

Comment:
Yes sure,

so this is useful if you have are storing different colorspaces using the same file format/bit depth for the same job.Example is show using wide gamut AcesCG and "standard" linear srgb primaries.
You can use the metadata to differentiate between what otherwise would have the exact same specs.